_includes/markdown/CSS.md

```diff
@@ -4,377 +4,578 @@ At 10up, we value content and the experience users will have reading it. We writ
 
-### CSS Specificity
+It’s okay to nest pseudo classes that represent state, for example:
 
-Stylesheets should go from less specific rules to highly specific rules. We want our selectors specific enough so that we don't rely on code order, but not too specific so that they can be easily overridden.
+```css
+/* block */
+.button {
 
-For that purpose, **classes** are our preferred selectors: pretty low specificity but high reusability.
+  /* ✅ state inside the block is fine */
+  &:hover {
+    /* styles here */
+  }
+}
 
-Avoid using `!important` whenever you can.
+/* ✅ element and modifier stay flat */
+.button__icon {
+  /* styles here */
+}
 
-Use [efficient selectors](https://csswizardry.com/2011/09/writing-efficient-css-selectors/).
+.button--primary {
+  /* styles here */
+}
+```
 
-Avoid:
+Not necessary to nest elements or modifiers:
 
 ```css
- div div header#header div ul.nav-menu li a.black-background {
-  background: radial-gradient(ellipse at center,  #a90329 0%,#8f0222 44%,#6d0019 100%);
+/* block */
+.button {
+
+  /* state inside the block is fine */
+  &:hover {
+    /* styles here */
+  }
+
+  /* ❌ element and modifier must not be nested inside the block */
+  & .button__icon {
+    /* styles here */
+  }
+
+  & .button--primary {
+    /* styles here */
+  }
 }
 ```
 
-### Inheritance
+### Avoid IDs
+
+Refrain from using IDs for styling purposes. IDs have high specificity, which makes them difficult to override and maintain. Instead, use classes for consistent and reusable styling.
+
+### Avoid \!important
+
+Reserve \!important for truly exceptional cases. Overusing \!important disrupts the cascade and makes debugging more difficult. Focus on writing clean, organized selectors instead of relying on this property to enforce styles.
 
-Fortunately, many CSS properties can be inherited from the parent. Take advantage of inheritance to avoid bloating your stylesheet but keep [specificity](#css-specificity) in mind.
+<h2 id="no-margins-components" class="anchor-heading">Do Not Use Margins on Reusable Components {% include Util/link_anchor anchor="no-margins-components" %}</h2>
 
-Avoid:
+Margins might look like one of the easiest CSS properties to manage, but they’re also a common source of layout headaches if not handled carefully.
+
+* Avoid margins on reusable components. Margins can lock components into specific contexts, making them harder to use in different layouts. If you add margins to a reusable component, you’re creating assumptions about where and how it will be placed.  
+* Design components with isolation in mind. Treat every component as if you have no idea what its future container will look like. Focus on the component's internal spacing and leave external spacing to the parent or container that uses it.
+
+### Example
+
+Let’s say you have a Card component. Avoid doing this:
```

Comment:
@dmtrmrv 

Now that I see this, I am wondering if we should add an example where we use relative values (i.e. `em`) to provide relative margin depending on context for vertical rhythm (i.e. headings)? I know you mention the lobotomized owl below, but this is an area I do see folks get hung up on - so a quick and easy example might be helpful. I 100% defer to you though! Simply a suggestion. If you feel this is an anti-pattern - please ignore.

Reply:
I've added this a bit further down